package.json

```diff
@@ -16,6 +16,7 @@
     },
     "homepage": "https://github.com/bedita/web#README",
     "scripts": {
+        "start": "(bin/cake server -H 0.0.0.0 -p 4545 & yarn develop --proxy 0.0.0.0:4545)",
```

Comment:
[see above] ... as a consequence better not add `bin/cake server` script here, since it's not the recommended for development